uber/api.py

```diff
@@ -1084,6 +1109,8 @@ def print_job_complete(self, job_ids=None, complete_all=False):
 
             if complete_all:
                 jobs = base_query.all()
+            elif not job_ids:
```

Comment:
We could just select all jobs if job_ids isn't passed through. My first inclination was to go this route since it's a bit more user-friendly, but if we feel like that's unnecessary I can take this out and simply have it pull all jobs if job_ids isn't specified.